### PR TITLE
Project style

### PR DESCRIPTION
**Pull request by kis87988**
This is our default project style.
Include folders, coding style guide, and commit message.

### CODE REVIEW

**Review by yifeili98 — DISMISSED** [review]

**Review by biankaiwen111 — DISMISSED** [review]

**Review by yifeili98 — DISMISSED** [review]

**Review by ak840206 — DISMISSED** [review]
All good!

**Review by yifeili98 — DISMISSED** [review]

**Review by yifeili98 — DISMISSED** [review]

**Review by yifeili98 — APPROVED** [review]

**Review by biankaiwen111 — APPROVED** [review]

**Review by emilylxf — COMMENTED** [review]
I have read all.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+1. max-line-length: 100
+
+Please also write docstring for contributor easy to follow.
+We follow Google docstring style(https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)
+
+Reference:
+
+1. https://www.python.org/dev/peps/pep-0008/
+2. http://google.github.io/styleguide/pyguide.html
```
Comment (hk-mp5a3) [inline README.md:65]: Can just use
```
1. https://www.python.org/dev/peps/pep-0008/
1. http://google.github.io/styleguide/pyguide.html
```

**Review by hk-mp5a3 — APPROVED** [review]
Approved. I haven't look into too many details since William has already reviewed it.

File: README.md (hunk 1 above)
Comment (kis87988) [reply README.md:65]: MD029

**Review by ak840206 — APPROVED** [review]
All good!

**Pull request merged**